lib/ansible/module_utils/gcp_utils.py

```diff
@@ -86,7 +86,7 @@ def get(self, url, body=None, **kwargs):
 
     def post(self, url, body=None, headers=None, **kwargs):
         if headers:
-            headers = self.merge_dictionaries(headers, self._headers())
+            headers = self._merge_dictionaries(headers, self._headers())
```

Comment:
@willthames This is already fixed here - https://github.com/ansible/ansible/blob/5d7cc993dde8ef32fafb606747be0ef03d31dc82/lib/ansible/module_utils/gcp_utils.py#L89

I am confused to see.

Reply:
Thanks @Akasurde - looks like I hadn't rebased in a few days (it's quite a new fix).

I'll keep this open as a WIP, as I've found some other issues.